.github/workflows/ci-tests.yml

```diff
@@ -148,13 +148,14 @@ jobs:
         run: task ci:reset
       - name: Run Cypress
         run: task ci:cypress
-      - name: Archive vidoes
+      - name: Archive videoes
+        if: always()
         uses: actions/upload-artifact@v3
         with:
           name: cypress-videos
           path: cypress/videos
       - name: Archive screenshots
-        if: ${{ failure() }}
+        if: always()
```

Comment:
I'm just wondering - is there a reason we now always archive screenshots? Even if the test doesn't fail?

Reply:
It is a bit vague for me now. But I believe I added it because otherwise we could not see the assets when test were failing.